static/spin/spin.css

```diff
@@ -0,0 +1,38 @@
+@keyframes spinner-line-fade-more {
```

Comment:
abrichr 9 days ago Member
Please move to static/css/lib

Reply:
Done

Reply:
https://github.com/MLDSAI/VisionTracing/pull/2/commits/9d7e98df8770e17db0981505f591ef45c5a48a5b